#! /bin/sh
patch -p1 -f $* < $0
exit $?

commit 11e876ad4
Author: Andrej Kacian <ticho@claws-mail.org>
Date:   Sat Dec 29 13:53:25 2018 +0100

    Remove use of %hhu format in managesieve for g_strdup_printf()

diff --git a/src/plugins/managesieve/sieve_prefs.c b/src/plugins/managesieve/sieve_prefs.c
index ec3713082..a398cfc61 100644
--- a/src/plugins/managesieve/sieve_prefs.c
+++ b/src/plugins/managesieve/sieve_prefs.c
@@ -579,7 +579,7 @@ void sieve_prefs_account_set_config(
 		enc_userid = g_base64_encode(config->userid, len);
 	}
 
-	confstr = g_strdup_printf("%c%c %s %c%hu %hhu %hhu %hhu %s",
+	confstr = g_strdup_printf("%c%c %s %c%hu %hu %hu %hu %s",
 			config->enable ? 'y' : 'n',
 			config->use_host ? 'y' : 'n',
 			config->host && config->host[0] ? config->host : "!",
